### ISSUE TITLE
Create designs for exporting information 'report style'

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/NorthernAustralia#51]
The users would like to have a functionality on the map which helps them in creating reports. 
Comments from user:
The Opportunity Analysis section is the part of the report that is generated from the map. We would like to be extract map pages into the report in a consistent style/format (as opposed to copying snapshots into a word document as I have document as I have done)  with fields/spaces to include text. This would include consistent way to show legends without having to show all the stuff that is currently in the workbench. Where time series data is used we would also like to be a be able to drop in charts that can be generated."

**Comment by AnaBelgun** [comment]
@philipgrimmett hi, it would be very helpful if we have some draft designs next week, so we are closer to making a decision on complexity, resourcing and time with our engineers and client.

**Comment by kring** [comment]
@AnaBelgun I think one thing we need to get clarification on is this statement:
> We would like to be extract map pages into the report

What does it mean to extract into a report? Alternate statements like "generate a report" or "create content to be pasted into a report" make sense to me. But "extract map pages into a report" sounds like they're expecting us to automate their Microsoft Word in order to push images/content into it, which I'm pretty sure is not a thing that a web site is allowed to do by web browser security.

**Comment by AnaBelgun** [comment]
@kring yes, that was clarified yesterday: no automated extraction and inclusion into their word or other templates. 
Some suggestions we discussed: 
- for example, when creating a map view which has: data selected, legends, feature information panel and graphs, find a way to export that into an image (or other way) which separates that information (legend, actual map, chart, feature info); if you just copy and paste screen image the info is unclear
- create a snapshot with the map view and a blanc box for free text although in the end the blanc text box is not so important, since people will include that in a word document (separately from the map)

**Comment by AnaBelgun** [comment]
We should link this to: https://github.com/TerriaJS/terriajs/issues/2820

**Comment by philipgrimmett** [comment]
Prototype: https://invis.io/35FXNA1CFGW#/280265196_workbench_With_Data

**Comment by kring** [comment]
Here are some things we can allow the user to do, if we want:
1. Download the map image (PNG or JPG)
2. Download legend images. For PNG or JPG legend images we'll just give them to the user as-is. For SVG images, we may want to allow the user to download the SVG _or_ download it as a PNG.
3. Set the size/resolution of the map. i.e. maybe you want to print at a higher resolution than your screen. Note that in this case there will be a "please while while we load some stuff" type thing going on. There may be a brief loading process even without letting the user specify the size/resolution.
4. Download a ZIP file containing all the resources, including the map image, legend images in multiple formats, text in markdown/HTML/PDF/Word format (not sure how the last two will go, but it might be doable or even easy), etc.
5. Print the entire exported view.
